Documentation/planning/arcade-powered-source-build/implementation-plan.md

```diff
@@ -1,47 +1,52 @@
-# Source-build 5.0 Implementation Plan
+# Arcade-powered source-build implementation plan
 
-To get each repo building with the new source-build 5.0 plan, [Arcade-Powered Source-Build](https://github.com/dotnet/source-build/tree/release/3.1/Documentation/planning/arcade-powered-source-build), the following stages will be accomplished.  Below, the repos are placed in a table representing the earliest dependencies are available and work can begin.  Stages can be completed in parallel, but stage 4 may depend on output from stage 6 from previous repos.  The following graph and table will be used to track progress for each repo.
+To get each repo building with the new source-build sustainability plan, [Arcade-Powered Source-Build](./README.md), each repo will go through several stages. Some stages have dependencies on upstream repos, so this process will be completed in dependency order with some exceptions.
 
-> ![](img/implementation-plan-graph.png)
-> [source (img/implementation-plan-graph.dgml)](img/implementation-plan-graph.dgml)
+Below, the repo status is in a graph to show the dependencies and make it easy to tell which repos are ready to work on. There's also a table, for searchable and more detailed status. These will both be kept up to date.
+
+> ![](img/implementation-plan-graph.svg)  
+> [source (img/implementation-plan-graph.dot)](img/implementation-plan-graph.dot)
 
 | Tier | Repo | Owner | (Stage 1)<br>Build from source - 5.0 | (Stage 2)<br>Input intermediate packages available | (Stage 3)<br>Merge patches & local build infra | (Stage 4)<br>Merge CI, generate official build | (Stage 5)<br>Validate & merge dev branch into 5.0 | (Stage 6)<br>Prebuilt baseline enforcement |
 | --- | --- | --- | :---: | :---: | :---: | :---: | :---: | :---: |
 | sbrp | Source-build-reference-packages | [Dan Seefeldt](https://github.com/dseefeld) | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ | |
-| Tools | sourcelink | [Tomas Matousek](https://github.com/tmat) | ✔️ | ✔️ | | | | |
+| Tools | sourcelink | [Tomas Matousek](https://github.com/tmat) | ✔️ | ✔️ | ✔️ | | | |
 | Tools | arcade | [Mark Wilkie](https://github.com/markwilkie) | ✔️ | | | | | |
+| 1 | *cssparser*<sup>1</sup> | ? | ✔️ | ✔️ | | | | |
+| 1 | *humanizer*<sup>1</sup> | ? | ✔️ | ✔️ | | | | |
+| 1 | *netcorecli-fsc*<sup>1</sup> | [Chris Rummel](https://github.com/crummel) | ✔️ | ✔️ | | | | |
+| 1 | *newtonsoft-json*<sup>1</sup> | [Chris Rummel](https://github.com/crummel) | ✔️ |  | | | | |
+| 1 | *newtonsoft-json901*<sup>1</sup> | [Chris Rummel](https://github.com/crummel) | ✔️ | ✔️ | | | | |
 | 1 | application-insights | [Reiley Yang](https://github.com/reyang) | ✔️ | ✔️ | | | | |
```

Comment:
I think application-insights may need to be maintained in the source-build repo as well.  Installer is depending on the 2.0 version which is a commit from Dec, 2017 and it's been that way since 2.1.  I'm not sure there'll be much appetite to move it forward.  I did file an issue in source-build about this, but I'm not sure it's our issue to drive.

Reply:
Looks like https://github.com/dotnet/source-build/issues/368. Yeah, if the commit needed by dotnet/installer is basically locked, it's pointless to force it for arcade-powered source-build. Marking as source-build and resetting ownership.